lib/src/chip/chip-row/chip-row.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, HostBinding, OnInit, Input } from '@angular/core';
+import { ChipComponent } from '../chip.component';
+import { FilterButtonComponent } from '../filter-button/filter-button.component';
+
+@Component({
+    selector: 'hc-chip-row',
+    templateUrl: './chip-row.component.html',
+    styleUrls: ['../chip.component.scss']
```

Comment:
You'll want to add encapsulation: ViewEncapsulation.None. We'll want to give user's the power to alter these